cardano-crypto-class/src/Cardano/Crypto/Hash/Class.hs

```diff
@@ -6,8 +6,7 @@
 module Cardano.Crypto.Hash.Class
   ( HashAlgorithm (..)
   , ByteString
-  , Hash
-  , getHash
+  , Hash(..)
```

Comment:
For consistency with the rest of the export list: add a space between `Hash` and `(..)`.